Add start date to Issues Table

scottdensmore opened the pull request:
* Add project in progress to the card for rendering
* Update the column widths to adjust layout for new column

scottdensmore committed:
Add start date to Issues Table

* Add project in progress to the card for rendering
* Update the column widths to adjust layout for new column

aloviedo commented on components/IssuesTable.tsx:
```diff
@@ -58,11 +59,20 @@ export default function IssuesTable(props: Props) {
+        accessor: 'project_in_progress_at',
+        className: 'w-32',
+        Cell: ({row, cell}: CellProps<Card, Date>) => (
+          <TargetDate targetDate={cell.value} />
```
We probably should renabe this component as ReportDate. Not a blocker.

scottdensmore replied:
Pushed a change to rename this to `ReportDate`.

scottdensmore committed:
Rename TargetDate to Report Date

Given we use this component for more than just target date, rename it to better describe the intended purpose.
